Mathlib/Analysis/SpecialFunctions/Log/ENNRealLog.lean

```diff
@@ -0,0 +1,228 @@
+/-
```

Comment:
The file should just be named `ENNReal`, because we're already in the `Log` subdirectory.